week_1_1.py

```diff
@@ -0,0 +1,154 @@
+"""
+
+TASK 1
+
+Write a class to represent a Cash Register.
+You class should keep the state of price total and purchased items
+
+Below is a starter code:
+------------------------
+1. you can add new variables and function if you want to
+2. you will NEED to add accepted method parameters where required.
+For example, method add_item probably accepts some kind of an item?..
+3. You will need to write some examples of how your code works.
+
+"""
```

Comment:
8/9 marks 🥳 

Overall, a lovely clear class with methods that don't do anything surprising. Looking back with a few weeks' more learning, it might be useful to think about how you could make some of this code a bit easier (particularly `total_item_details`), but it fulfils the requirements and gets all the marks for that.

-1 mark for `print`ing where `return`s would be more useful.